src/Backends/Banshee.GStreamer/libbanshee/banshee-gst.h

```diff
@@ -66,6 +66,7 @@ typedef void (* BansheeLogHandler) (BansheeLogType type, const gchar *component,
 
 MYEXPORT void
 gstreamer_initialize (gboolean debugging, BansheeLogHandler log_handler);
+void      gstreamer_deinitialize ();
```

Comment:
I think this will need MYEXPORT as above for the initialize function - I don't think its lack will bother us on GNU systems but maybe the windows folk need it?